gadget-amd64.yaml

```diff
@@ -18,7 +18,7 @@ volumes:
         type: EF,C12A7328-F81F-11D2-BA4B-00A0C93EC93B
         filesystem: vfat
         filesystem-label: system-boot
-        size: 256M
+        size: 260M
```

Comment:
To allow upgrades of gadget in older images, you could do:
```
min-size: 256M
size: 260M
```
This will specify a valid range of sizes for upgrades, but new images will use the `size` field.

Reply:
Thanks Alfonso! I've updated the commit.